content/R_Software_Setup.md

```diff
@@ -0,0 +1,136 @@
+
+Copy and paste the code
+-----------------------
+
+Click in the window and paste in the code from below:
+
+``` r
+version$version.string
+```
+
+Verify your installed R version
+-------------------------------
+
+Ideally you should be running the latest stable release (v3.4.1 as of June 2017). If you have an older version, please install the newest version using the instructions above.
+
+Here is what my R version looks like.
+
+    ## [1] "R version 3.4.0 Patched (2017-06-09 r72776)"
+
+Add packages and GRAN
+=====================
+
+Install general packages
+------------------------
+
+Install packages available for all R users from [CRAN](https://cran.r-project.org/) by pasting the following code and hitting `ENTER`.
+
+``` r
+install.packages(c("dplyr","dataRetrieval","ggplot2","tidyr","RColorBrewer","EGRET", "rmarkdown", "geoknife"))
+```
+
+If you see
+
+![Dialog box behind](../static/img/personal_library_dialog.png#inline-img "dialog box behind")
+
+That's OK! There’s a dialog box hiding behind RStudio asking if you want to create a personal folder in your Documents folder. Click *Yes*. It will look something like this when done:
+
+![Console output](../static/img/general_pkg_output.png#inline-img "console output")
+
+Add GRAN as a default repository
+--------------------------------
+
+Modifying the .Rprofile file will tell your R environment to always look in the GRAN repository when installing packages and updating packages. This is the preferred method because of the ability to automatically get GRAN updates. Paste the following code into the Rstudio console to update your .Rprofile:
+
+``` r
+rprofile_path = file.path(Sys.getenv("HOME"), ".Rprofile")
+write('\noptions(repos=c(getOption(\'repos\'),
+    CRAN=\'https://cloud.r-project.org\',
+    USGS=\'https://owi.usgs.gov/R\'))\n',
+      rprofile_path, 
+      append =  TRUE)
+
+cat('Your Rprofile has been updated to include GRAN.
+    Please restart R for changes to take effect.')
+```
+
+Restart RStudio (close and open). Now, you'll automatically have access to USGS packages and updates.
+
+Package best practices
+----------------------
+
+Now that you have installed R and some basic packages, you should get in a good habit of updating them regularly in case the authors fix bugs or add new features. It's best to do this on a weekly basis so that your code is never outdated. To update, click the icon in your packages tab. You can update them all at once or individually.
+
+![Update your packages](../static/img/update_pkgs.png#inline-img "update your packages")
+
+Optional: Install Rtools
+========================
+
+Rtools is used in package development, which is an advanced skill, but one which you might be doing as you become more experienced. It would be handy to have Rtools installed. You will need to install this using admin privileges. Install from [here](https://cran.r-project.org/bin/windows/Rtools/).
```

Comment:
Yeah, I'd mention it here too just because there's no equivalent for Macs, I could see that being confusing (whereas everything else is pretty similar)